### PR TITLE
script-food.rpy (Plus a few other things)

### PR DESCRIPTION
**Pull request by Ulvsblakk**
It's been an astronomically long time since I've made a pull request with the mod.
If I did everything right, awesome. If not I'll fix it. 

Still need some dialogue and expressions to go along with it. Will be adding changes as I go along. If you guys want to put some in your self, feel free. Potato said a scroll-able menu can fit tons of options. I also wouldn't mind multiple dialogues for each food item.

WARNING: The only problem I can see as of now, is that the menu icon for this shows up under Goodbye. If that's not an issue, great. If so, again I'll figure out how to fix it. Thank you everyone.

### CODE REVIEW

File: Monika After Story/game/event-handler.rpy [hunk 1]
```diff
@@ -212,6 +212,8 @@ init -900 python in mas_ev_data_ver:
     verify_event_data(store.persistent._mas_mood_database)
     verify_event_data(store.persistent._mas_story_database)
     verify_event_data(store.persistent._mas_apology_database)
+    verify_event_data(store.peristent._mas_food_database)
```
Comment (multimokia) [inline Monika After Story/game/event-handler.rpy]:
```suggestion
    verify_event_data(store.persistent._mas_food_database)
```

File: Monika After Story/game/script-food.rpy [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    return _return
+
+
+
+init 5 python:
+    addEvent(Event(persistent._mas_food_database,'mas_food_coffee',prompt='Coffee',category=[store.mas_food.TYPE_DRINK],unlocked=True,),code='FOO')
+
+label mas_food_coffee:
+    m 1eka "That sounds wonderful."
+    m "I really miss having coffee."
```
Comment (Rai99) [inline Monika After Story/game/script-food.rpy]: Should account for if the player gave her coffee and she can make her own coffee in the morning.

File: Monika After Story/game/script-food.rpy [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    m "I would always have a cup before and after school."
+    m "Maybe even sometimes during class"
```
Comment (Rai99) [inline Monika After Story/game/script-food.rpy]: Missing a period.

File: Monika After Story/game/script-food.rpy [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+    m "I wish I could figure out a way to get some in here."
+    m "Who knows, maybe you could share a cup with me [player]"
```
Comment (Rai99) [inline Monika After Story/game/script-food.rpy]: Add a comma after "me". 

Missing a period.

File: Monika After Story/game/script-food.rpy [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+    return
+
+init 5 python:
+    addEvent(Event(persistent._mas_food_database,'mas_food_water',prompt='Water',category=[store.mas_food.TYPE_DRINK],unlocked=True,),code='FOO')
+label mas_food_water:
+    m "I'm really happy to hear that [player]!"
```
Comment (Rai99) [inline Monika After Story/game/script-food.rpy]: Add a comma after "that".

File: Monika After Story/game/script-food.rpy [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+    m "Water is probably the healthiest thing that you could drink."
+    m "Seeing how we can only live for a few days without it."
+    m "Just promise you get plenty of drink each day."
```
Comment (Rai99) [inline Monika After Story/game/script-food.rpy]: Replace "of" with "to".

File: Monika After Story/game/script-food.rpy [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+    m "I don't want you getting sick because you didn't drink any."
+    m "Even if it's just a glass or two, do it for me."
+    return
+init 5 python:
+    addEvent(Event(persistent._mas_food_database,'mas_food_milk',prompt='Milk',category=[store.mas_food.TYPE_DRINK],unlocked=True,),code='FOO')
+label mas_food_milk:
+    m "You know, I've never really drank a lot milk before."
+    m "It wasn't because I didn't like it either."
+    m "I just never really thought about it, unless I put it in coffee, or mixed chocolate in with it."
+    m "I guess I could learn to love it if you do [player]."
```
Comment (Rai99) [inline Monika After Story/game/script-food.rpy]: Add a comma after "do".

**Comment by Ulvsblakk** [comment]
Busy atm. Will fix later.

**Comment by kaido1224** [comment]
Be mindful of what sort of food you add, since Monika is a vegetarian and all.

**Comment by Rai99** [comment]
I thought this was about the player?

**Comment by Ulvsblakk** [comment]
It is about the player

**Comment by Ulvsblakk** [comment]
Alright guys. I made it so it shows up as a regular pool topic, and not on the main menu. I'm going to close this pull request and open a new one, as making it a pool topic, there's no use in creating a new "Foods" script, I just implement that in the topic in script-topics. I'll add dialogue and expressions and make a new pull request. Thanks for the feed back.

**Comment by multimokia** [comment]
Wait, you *can* use the foods script, since there's going to be a bunch of topics for it, right? (One for each type of food) So keeping it all in the food script would work.

**Comment by Rai99** [comment]
If there are going to be hundreds of food and drink topics (which if this really takes off, there probably will be), I think it would be best to make a new file for it so the regular topics file doesn't explode in size.

**Pull request closed without merging**